.gitignore

```diff
@@ -117,4 +117,7 @@ ENV/
 
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
\ No newline at end of file
+yarn-error.log*
+
+# generated frontend
+staticfiles*
```

Comment:
@ericsandine -- can you confirm this is ok to ignore? I only saw the compiled output of `yarn run build` in that folder.

Reply:
Ah yeah you should be able to ignore that, should just be built.